### PR TITLE
adds pass-suppress-confirmations option

### PR DESCRIPTION
**Pull request by titibandit**
Addresses issue #52.

### CODE REVIEW

File: pass.el [hunk 1]
```diff
@@ -157,7 +163,8 @@ Similar to `save-excursion' but only restore the point."
 (defun pass-quit ()
   "Kill the buffer quitting the window."
   (interactive)
-  (when (y-or-n-p "Kill all pass entry buffers? ")
+  (when (or pass-suppress-confirmations
+             (y-or-n-p "Kill all pass entry buffers? "))
```
Comment (doolio) [inline pass.el:167]: Should we use `yes-or-no-p` here for consistency with the other prompts or use `y-or-n-p` in the other two?

**Comment by doolio** [comment]
cc: @DamienCassou what do you think?

File: pass.el (hunk 1 above)
Comment (DamienCassou) [reply pass.el:167]: I think `y-or-n-p` should be used each time the operation is not dangerous (killing a saved buffer is not dangerous, deleting a file is).

**Review by DamienCassou — APPROVED** [review]

File: pass.el (hunk 1 above)
Comment (doolio) [reply pass.el:167]: OK, then I think we can merge this change.

**Comment by DamienCassou** [comment]
Thank you @titibandit for the PR and @doolio for the review!

**Pull request merged**